src/terminal/terminal/display.h

```diff
@@ -340,6 +340,25 @@ void guac_terminal_display_flush(guac_terminal_display* display);
 void guac_terminal_display_dup(guac_terminal_display* display, guac_user* user,
         guac_socket* socket);
 
+/**
+ * Initializes and syncs the current terminal display state for all joining
+ * users associated with the provided socket, sending the necessary instructions
+ * to completely recreate and redraw the terminal rendering over the given
+ * socket.
+ *
+ * @param display
+ *     The terminal display to sync to the users associated with the provided
+ *     socket.
+ *
+ * @param client
+ *     The client whose users are joining.
+ *
+ * @param socket
+ *     The socket over which any necessary instructions should be sent.
+ */
+void guac_terminal_display_dup_batch(
+        guac_terminal_display* display, guac_client* client, guac_socket* socket);
```

Comment:
Same here - I think there's no need to maintain the old version of this function, unless it has some utility internally within `guac_terminal`. It's not part of the public API.

Reply:
Oops, yeah I'll get rid of this. I had mistakenly thought it was part of the public API.